guide/designing-products/usage-life-cycle.md

```diff
@@ -36,9 +36,11 @@ Illustration sources
 # Usage life cycle
 {:.no_toc}
 
-We have all mastered different skills and tools that we care about or use frequently. Many times we forget what the initial struggles were once we reach a certain comfort level. A similar change happens when we develop products. We learn and internalize different mechanisms and logic, and what was once hard becomes intuitive, and we end up building things with overwhelming learning curves. So it's a healthy habit to regularly take a step back and look through users' eyes at different levels of experience with your product.
+As we develop new skills, things that are difficult at first eventually become intuitive to us. So when we develop products, it’s a good idea to regularly step back and look at the products we create through users' eyes on that same journey.
 
-The following six sections outline different phases of a typical usage life cycle, from somebody who has never heard of your product to them becoming a power user. It is helpful to think through each phase and consider what the current user needs are, what their understanding of bitcoin and your product likely is, and how you can help them successfully navigate.
+The following six sections outline a typical usage life cycle, from never having heard of your product to becoming one of its power users.
```

Comment:
```suggestion
The following six phases outline a typical usage life cycle, from never having heard of your product to becoming one of its power users.
```
We use sections to refer to parts of the guide, so changing this to phases, which is also used in the following paragraph.